### DISCUSSION TITLE
New to HASPone and HASP - wait for new Wiki instructions? Or can I start building the original?

### DISCUSSION THREAD

**Discussion by rkhanso** [HASwitchPlate/HASPone#84] [General]
Hello,
I'd like to build a few of these for my home.  I think having one on each level of the home would be nice. Not quite ready to start building, but hoping to in the next 2-3 months.

I see there's a new version out, and the website says: "While the new hardware build docs are being re-written, check the original project site here."

Can I start building the original device (is the hardware the same?) or should I wait until new instructions come out? 

Thanks.

**Comment by xHirscHx** [comment]
https://www.youtube.com/watch?v=zi784hr69QA
https://www.youtube.com/watch?v=hTb49n4butI
 Same hardware new firmware  works great got 5
